arangodb-net-standard/GraphApi/PostGraphEdgeBody.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ArangoDBNetStandard.GraphApi
+{
+    public class PostGraphEdgeBody
```

Comment:
I don't see this class being used anywhere, though I may miss something. Can you confirm?